### DISCUSSION TITLE
add an option for this type of cursor trail size

### DISCUSSION THREAD

**Discussion by Weeblino** [ppy/osu#27854] [Ideas] [answered]
in lazer this type of cursor trail doesn't seem to change it's size regardless of the gameplay cursor size, it bothers me a lot. i like very thin cursor trail. it would be cool to add an option to fix this
![lazer](https://github.com/ppy/osu/assets/166963554/bc30a075-e608-40a8-b0de-12a17ac21610)
![stable](https://github.com/ppy/osu/assets/166963554/fa07ab58-0fd3-4a34-95d7-eae3da6182db)

**Comment by Stoppedpuma** [comment, marked as answer]
> this type of cursor trail doesn't seem to change it's size regardless of the gameplay cursor size

It does change, this is more likely just it not matching the size of stable 1:1. Current solutions are editing the size of the cursor trail or just making the gameplay cursor size smaller.
